samza-log4j/src/main/java/org/apache/samza/logging/log4j/StreamAppender.java

```diff
@@ -279,8 +285,17 @@ protected Config getConfig() {
     return config;
   }
 
+  private void addDropProducerErrorsConfig() {
+    Map<String, String> newConfig = new HashMap<>();
+    if (config.get(DROP_PRODUCER_ERRORS) == null) {
+      newConfig.put(DROP_PRODUCER_ERRORS, "true");
+      config = new MapConfig(ImmutableList.of(config, newConfig));
```

Comment:
remove this method and inline this inside `getConfig` so that all config wirings happen in a single place

Reply:
Got it. Done.